packages/playback-core/src/errors.ts

```diff
@@ -17,6 +29,8 @@ export class MediaError extends Error {
 
   name: string;
   code: number;
+  public muxCode?: number;
```

Comment:
NOTE: plausibly want to drop `errorCategory` and `muxCode`, but keeping for now just in case, since error context, message, and code are information lossy.